build.gradle.kts

```
@@ -308,10 +310,12 @@ tasks {
 	}
 
 	javadoc {
-		javadocTool.set(project.javaToolchains.javadocToolFor {
-			// Create Javadoc with at least Java 17 to get the latest features, e.g. search bar
-			languageVersion.set(JavaLanguageVersion.of(maxOf(17, targetJavaVersion.majorVersion.toInt())))
-		})
+		if (releaseBuild) {
+			javadocTool.set(project.javaToolchains.javadocToolFor {
+				// Create Javadoc with at least Java 17 to get the latest features, e.g. search bar
+				languageVersion.set(JavaLanguageVersion.of(maxOf(17, targetJavaVersion.majorVersion.toInt())))
```

This can be simplified, as it now only happens during release:

```suggestion
				languageVersion.set(JavaLanguageVersion.of(17))
```

---

Good point, I just hard-coded 20.